samples/VanillaJSTestApp2.0/app/onPageLoad/ui.js

```diff
@@ -9,7 +9,8 @@ const profileDiv = document.getElementById("profile-div");
 function showWelcomeMessage(account) {
     // Reconfiguring DOM elements
     cardDiv.style.display = 'initial';
-    welcomeDiv.innerHTML = `Welcome ${account.name}`;
+    username = account.username;
```

Comment:
Can we move this to where we handle the response in `auth.js` to make it more clear how we identify the user? I'm afraid this will get looked over and people will be confused how to call `getAccountByUsername`

Reply:
done.